operatorcourier/api.py

```diff
@@ -19,14 +19,16 @@
 logger = logging.getLogger(__name__)
 
 
-def build_and_verify(source_dir=None, yamls=None, ui_validate_io=False):
+def build_and_verify(source_dir=None, yamls=None, ui_validate_io=False, repository=None):
     """Build and verify constructs an operator bundle from
     a set of files and then verifies it for usefulness and accuracy.
 
     It returns the bundle as a string.
 
     :param source_dir: Path to local directory of yaml files to be read.
     :param yamls: List of yaml strings to create bundle with
+    :param ui_validate_io: TODO
```

Comment:
Let's leave documentation for this parameter for a separate pr please.

Reply:
Sure. Am I supposed to remove this line?

Reply:
that is what I mean, yes.